src/server/pfs/server/api_server.go

```diff
@@ -530,6 +530,32 @@ func (a *apiServer) GetFileTAR(request *pfs.GetFileRequest, server pfs.API_GetFi
 	})
 }
 
+// GetFile implements the protobuf pfs.GetFile RPC
+func (a *apiServer) GetFile(request *pfs.GetFileRequest, server pfs.API_GetFileServer) (retErr error) {
+	func() { a.Log(request, nil, nil, 0) }()
+	defer func(start time.Time) { a.Log(request, nil, retErr, time.Since(start)) }(time.Now())
+	return metrics.ReportRequestWithThroughput(func() (int64, error) {
+		ctx := server.Context()
+		src, err := a.driver.getFile(ctx, request.File)
```

Comment:
This seems kind of heavyweight for `api_server.go` - maybe it's fine because `GetFile` can't be done in an existing transaction anyway so an RPC boundary doesn't hurt.